src/al.jl

```diff
@@ -120,6 +132,7 @@ function al(nlp :: AbstractNLPModel; mu :: Real = eltype(nlp.meta.x0)(10.0),
     end
   end
 
-  return GenericExecutionStats(status, nlp, solution = al_nlp.x[1:nlp.meta.nvar-ns], objective = obj(nlp, al_nlp.x), dual_feas = normgp, primal_feas = normcx,
-                               iter = iter, elapsed_time = el_time)
+  return GenericExecutionStats(status, nlp, solution = al_nlp.x,
+                               objective = obj(nlp, al_nlp.x), dual_feas = normgp, primal_feas = normcx,
```

Comment:
Maybe the augmented Lagrangian struct could keep the value of the original obj around so it doesn't have to be recomputed?

Reply:
I think `S.objective` can reused removing the aug. penalty part.

Reply:
But I think it's better in a new PR.

Reply:
That's right. You're already doing something similar on line 35, so might as well do it here, no?

Reply:
Ok. It's updated.